src/Lesson1/Order.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+export default function Order({ OrderType }) {
+  const [order, setOrder] = useState(0);
+  function orderOne() {
+    setOrder(order + 1);
+  }
+  return (
+    <>
+      <li>
+        {OrderType}: {order}
+        <RestaurantButton orderOneProps={orderOne} />
+      </li>
+    </>
+  );
```

Comment:
Good - the button is a separate component - but in effect you give the button something to do when it is clicked.